daemons/dss-scalability-test/requirements.txt

```diff
@@ -0,0 +1,49 @@
+hca >= 3.2.0
```

Comment:
Why is this requirements file not a link to the one in the parent directory? If it's a superset of the one in the parent directory, then let's make it include that one with "-r ../../requirements.txt".

Reply:
agreed, done